addon/components/primitives/frost-list-item-unlabeled-property.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember'
```

Comment:
I think we could genericize this as `property` and just react to the presence/absence of a label